src/dvr/dvr_rec.c

```diff
@@ -169,6 +170,18 @@ cleanup_filename(char *s, dvr_config_t *cfg)
       s[i] = '_';
   }
 
+  if(cfg->dvr_windows_compatible_filenames) {
+    //trim trailing spaces and dots
+    for (i = len2 - 1; i >= 0; i--) {
+      if((s[i] == ' ') || (s[i] == '.')) {
```

Comment:
Could you rewrite this condition as opposite:

if (!wrong_char)
  break;
s[i] = '\0';

It's more clean.